Makefile

```diff
@@ -152,14 +152,20 @@ website-install:
 website-start:
 	@npm start --prefix website/
 
+test-database-up:
+	make DOCKER_ARGS='-d' -C testing/dbtest/docker database-up
+
+test-database-down:
+	make -C testing/dbtest/docker clean
+
 test-ci: install-go
 	~/.go/bin/go test ./... -v $(TESTARGS) -timeout 120m
 
 test-sql:
 	$(MAKE) -C internal/db/sqltest/ test
 
-test: 
-	~/.go/bin/go test ./... -timeout 30m
+test:
```

Comment:
Can we integrate `test-database-up` and `test-database-down` into this target so that if we want to run all tests locally we get the same behavior?

Reply:
Yea I can do that. I wasn't sure if people tend to run `make test` frequently (I would guess not given how long it tends to take), but with these changes the workflow I was envisioning would be to run `make test-database-up` once, and then be able to run `make test` multiple times without needing to teardown and recreate the test database. The only time you really need to recreate the test database is if new migrations are added.

But this can just as easily be done by using `make test-database-up` and calling `go test` directly.

Reply:
I looks like it is still the case that `./...` stays within a single module. Should we be explicitly running the tests for `api/` and `sdk/` in CI?